# 2022-08-04T00:58:27.730Z:
test_status: ERROR
platform_error {
  error_detail {
    summary {
      namespace {
        namespace: "com.google.testing.platform"
      }
      error_code: 2002
      error_name: "PLUGIN_ERROR"
      error_classification: "TEST"
      error_message: "Exception thrown during onBeforeAll invocation of plugin com.google.testing.platform.plugin.android.AndroidDevicePlugin."
      stack_trace: "com.google.testing.platform.api.plugin.PluginException: Exception thrown during onBeforeAll invocation of plugin com.google.testing.platform.plugin.android.AndroidDevicePlugin.\r\n\tat com.google.testing.platform.plugin.PluginLifecycleKt.invokeOrThrow(PluginLifecycle.kt:216)\r\n\tat com.google.testing.platform.plugin.PluginLifecycleKt.invokeOrThrow$default(PluginLifecycle.kt:205)\r\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1.invoke(PluginLifecycle.kt:92)\r\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1.invoke(PluginLifecycle.kt:88)\r\n\tat com.google.testing.platform.core.telemetry.common.noop.NoopDiagnosticsScope.recordEvent(NoopDiagnosticsScope.kt:35)\r\n\tat com.google.testing.platform.core.telemetry.TelemetryKt.recordEvent(Telemetry.kt:105)\r\n\tat com.google.testing.platform.core.telemetry.TelemetryKt.recordEvent$default(Telemetry.kt:98)\r\n\tat com.google.testing.platform.plugin.PluginLifecycle.onBeforeAll(PluginLifecycle.kt:88)\r\n\tat com.google.testing.platform.executor.SingleDeviceExecutor$execute$4.invoke(SingleDeviceExecutor.kt:86)\r\n\tat com.google.testing.platform.executor.SingleDeviceExecutor$execute$4.invoke(SingleDeviceExecutor.kt:86)\r\n\tat com.google.testing.platform.executor.SingleDeviceExecutor.runUnlessCancelled(SingleDeviceExecutor.kt:105)\r\n\tat com.google.testing.platform.executor.SingleDeviceExecutor.execute(SingleDeviceExecutor.kt:86)\r\n\tat com.google.testing.platform.RunnerImpl.run(RunnerImpl.kt:108)\r\n\tat com.google.testing.platform.server.strategy.NonInteractiveServerStrategy$run$4.invoke(NonInteractiveServerStrategy.kt:80)\r\n\tat com.google.testing.platform.server.strategy.NonInteractiveServerStrategy$run$4.invoke(NonInteractiveServerStrategy.kt:79)\r\n\tat com.google.testing.platform.core.telemetry.common.noop.NoopDiagnosticsScope.recordEvent(NoopDiagnosticsScope.kt:35)\r\n\tat com.google.testing.platform.core.telemetry.TelemetryKt.recordEvent(Telemetry.kt:66)\r\n\tat com.google.testing.platform.server.strategy.NonInteractiveServerStrategy.run(NonInteractiveServerStrategy.kt:79)\r\n\tat com.google.testing.platform.main.MainKt$main$4.invokeSuspend(Main.kt:67)\r\n\tat kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)\r\n\tat kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)\r\n\tat kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:274)\r\n\tat kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)\r\n\tat kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)\r\n\tat kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)\r\n\tat kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)\r\n\tat kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)\r\n\tat com.google.testing.platform.main.MainKt.main(Main.kt:66)\r\n\tat com.google.testing.platform.main.MainKt.main$default(Main.kt:34)\r\n\tat com.google.testing.platform.main.MainKt.main(Main.kt)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat com.google.testing.platform.launcher.Launcher.main(Launcher.java:149)\r\nCaused by: kotlinx.coroutines.TimeoutCancellationException: Timed out waiting for 120000 ms\r\n\tat kotlinx.coroutines.TimeoutKt.TimeoutCancellationException(Timeout.kt:186)\r\n\tat kotlinx.coroutines.TimeoutCoroutine.run(Timeout.kt:156)\r\n\tat kotlinx.coroutines.EventLoopImplBase$DelayedRunnableTask.run(EventLoop.common.kt:497)\r\n\tat kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:274)\r\n\tat kotlinx.coroutines.DefaultExecutor.run(DefaultExecutor.kt:69)\r\n\tat java.base/java.lang.Thread.run(Thread.java:829)\r\n"
    }
    cause {
      summary {
        error_message: "Timed out waiting for 120000 ms"
        stack_trace: "kotlinx.coroutines.TimeoutCancellationException: Timed out waiting for 120000 ms\r\n\tat kotlinx.coroutines.TimeoutKt.TimeoutCancellationException(Timeout.kt:186)\r\n\tat kotlinx.coroutines.TimeoutCoroutine.run(Timeout.kt:156)\r\n\tat kotlinx.coroutines.EventLoopImplBase$DelayedRunnableTask.run(EventLoop.common.kt:497)\r\n\tat kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:274)\r\n\tat kotlinx.coroutines.DefaultExecutor.run(DefaultExecutor.kt:69)\r\n\tat java.base/java.lang.Thread.run(Thread.java:829)\r\n"
      }
    }
  }
}
